### ISSUE TITLE
OpenSSL 1.1.0 support

### ISSUE THREAD

**Issue by hex2a** [richardkiss/pycoin#261]
Is pycoin compatible with openssl 1.1.0?

when using libssl1.1 `1.1.0g-2` (Debian testing), this error occurs:

```
AttributeError: /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1: undefined symbol: BN_init
```

**Comment by mflaxman** [comment]
I get this on OpenSSL 1.1 using Tails as well

**Comment by hex2a** [comment]
I'm submitting a PR, which allows setting the library path via the env variable `PYCOIN_LIBCRYPTO_PATH`, so openssl 1.0.2 can be used if it is installed.

e.g. `PYCOIN_LIBCRYPTO_PATH=libcrypto.so.1.0.2`

**Comment by richardkiss** [comment]
Fixed in fabf92f3717fb3765034a28f87b4eaa641048092
